index.js

```diff
@@ -1,8 +1,29 @@
-var https = require('https');
+var request = require('request');
 var readline = require('readline'),
     rl = readline.createInterface(process.stdin, process.stdout);
 const fs = require('fs');
 
+if (process.argv.length <= 2) {
+    console.log('Usage: master password needs to be passed on command line');
+    process.exit(-1);
+}
+var masterPassword = process.argv[2];
+
+var baseUrl = 'https://calendar.knilxof.org/api/v2';
+var loginRequest = request.defaults({
+    url: baseUrl + '/login',
+    json: true
+}), createUserRequest = request.defaults({
+    url: baseUrl + '/users',
+    json: true
+}), createGroupRequest = request.defaults({
+    url: baseUrl + '/groups',
+    json: true
+}), addToGroupRequest = request.defaults({
+    baseUrl: baseUrl + '/groups/',
+    json: true
+});
```

Comment:
Actually I would have created the last 3 after we got 2 tokens (master for createUser and admin for the 2 others) so that we can set the tokens in the defaults.